File: openapi/code_samples/csharp/branches/create.cs
var client = new RestClient("https://staging-api.tizo.co/api/v1/vendors/stores/:idStore/branches/");
client.Timeout = -1;
var request = new RestRequest(Method.POST);
request.AddHeader("Authorization", "eyJhbGciOiJIUzUxMiJ9.eyJyb2xlIjoidmVuZG9yIiwidXNlcm5hbWUiOiJ2X2F4ZWxfOSIsImlhdCI6MTY1ODE2NDM4OCwiZXhwIjoxNjU4NzY5MTg4fQ.vvLtYB4O73futNvwzTZt2LhFzNV--XvMfTT5-62YWtxX-9CY6X4-UtIVDkCR_MX4YqX83AcCXiquHAb8uNHIjw");
request.AddHeader("Content-Type", "application/json");
var body = @"{" + "\n" +@"  ""name"": ""string""," + "\n" +@"  ""idMunicipality"": 0," + "\n" +@"  ""addressGeo"": ""string""," + "\n" +@"  ""addressHuman"": ""string""," + "\n" +@"  ""latitude"": 0," + "\n" +@"  ""longitude"": 0," + "\n" +@"  ""active"": true," + "\n" +@"  ""vacation"": true" + "\n" +@"}";
request.AddParameter("application/json", body,  ParameterType.RequestBody);
IRestResponse response = client.Execute(request);
Console.WriteLine(response.Content);

File: openapi/code_samples/python/branches/create.py
import requests
import json

url = "https://staging-api.tizo.co/api/v1/vendors/stores/:idStore/branches/"

payload = json.dumps({
  "name": "string",
  "idMunicipality": 0,
  "addressGeo": "string",
  "addressHuman": "string",
  "latitude": 0,
  "longitude": 0,
  "active": True,
  "vacation": True
})
headers = {
  'Authorization': 'eyJhbGciOiJIUzUxMiJ9.eyJyb2xlIjoidmVuZG9yIiwidXNlcm5hbWUiOiJ2X2F4ZWxfOSIsImlhdCI6MTY1ODE2NDM4OCwiZXhwIjoxNjU4NzY5MTg4fQ.vvLtYB4O73futNvwzTZt2LhFzNV--XvMfTT5-62YWtxX-9CY6X4-UtIVDkCR_MX4YqX83AcCXiquHAb8uNHIjw',
  'Content-Type': 'application/json'
}

response = requests.request("POST", url, headers=headers, data=payload)

print(response.text)
